com.hack_computer.Clubhouse.json

```diff
@@ -0,0 +1,276 @@
+{
+    "app-id": "com.hack_computer.Clubhouse",
+    "add-extensions": {
+        "com.hack_computer.Clippy.Extension": {
+            "version": "master",
+            "directory": "clippy",
+            "no-autodownload": false,
+            "autodelete": true
+        }
+    },
+    "branch": "master",
+
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+
+    "command": "eos-clubhouse",
+
+    "finish-args": [
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--env=GNOTIFICATION_BACKEND=gtk",
+        "--env=GSETTINGS_SCHEMA_DIR=/run/host/usr/share/glib-2.0/schemas/",
+        "--env=GTK3_MODULES=/app/clippy/lib/libclippy-module.so",
+        "--filesystem=/var/lib/AccountsService/icons/:ro",
+        "--filesystem=/var/lib/flatpak/exports/share/applications:ro",
+        "--filesystem=/var/lib/flatpak/exports/share/icons:ro",
+        "--filesystem=host",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=~/.local/share/flatpak",
+        "--nofilesystem=~/.local/lib/python3.5",
+        "--own-name=com.endlessm.GameStateService",
+        "--own-name=com.hack_computer.Clubhouse",
+        "--own-name=com.hack_computer.GameStateService",
+        "--own-name=com.hack_computer.HackSoundServer",
+        "--own-name=com.hack_computer.HackToolbox",
+        "--share=ipc",
+        "--socket=pulseaudio",
+        "--socket=session-bus",
```

Comment:
you're going to need specific justification for snooping everything here rather than specific address'

Reply:
We talk to "random" apps using the `org.gtk.Actions` or `org.gtk.Application` interfaces, that's why we need this. We can remove this and just add the dbus names for the applications, but we've a lot of different quests with an app associated and that list will grow in the future. And also we want to allow third party quests, linked to third party apps, so I think that we can't control the full list of apps this app will talk to. Thinking about quests defined on user home, like `./local` or other common place, to allow to distribute `quests` independently of the clubhouse app.

Here's the code that creates those dbus proxies for apps in quests: https://github.com/endlessm/clubhouse/blob/master/eosclubhouse/system.py#L526-L616

Reply:
Yeah, that's legit (and it's what I thought but we need to make it clear in review)